crates/valence_inventory/src/player_inventory.rs

```diff
@@ -0,0 +1,24 @@
+use std::ops::RangeInclusive;
+
+pub struct PlayerInventory;
+
+impl PlayerInventory {
+    pub const SLOT_OFFHAND: u16 = 45;
+    pub const SLOT_HEAD: u16 = 5;
+    pub const SLOT_CHEST: u16 = 6;
+    pub const SLOT_LEGS: u16 = 7;
+    pub const SLOT_FEET: u16 = 8;
+    pub const SLOTS_CRAFT_INPUT: RangeInclusive<u16> = 1..=4;
+    pub const SLOT_CRAFT_RESULT: u16 = 0;
+    pub const SLOTS_HOTBAR: RangeInclusive<u16> = 36..=44;
+    pub const SLOTS_MAIN: RangeInclusive<u16> = 9..=44;
+    pub const MAIN_SIZE: u16 = 36;
```

Comment:
I merged in a different PR earlier that added similar constants (#588), would you mind adding a revert commit to this PR?

```
git revert bbb8aa5
```